Refresh.GameServer/Types/Levels/GameLevel.cs

```diff
@@ -25,8 +25,14 @@ public partial class GameLevel : IRealmObject, INeedsPreparationBeforeSerializat
 
     [XmlElement("firstPublished")] [JsonProperty] public long PublishDate { get; set; } // unix seconds
     [XmlElement("lastUpdated")] [JsonProperty] public long UpdateDate { get; set; }
+    
+    [XmlElement("minPlayers")] [JsonProperty] public int MinPlayers { get; set; }
```

Comment:
Adding [JsonProperty] will only work for ApiV2 (which will be removed soon), you have to add it to `ApiGameLevelResponse` and map it